migrations/Makefile

```diff
@@ -2,6 +2,7 @@ SHELL:=/bin/bash
 PROJECT_NAME ?= kotsadm-migrations
 POSTGRES_ALPINE_TAG ?= 10.19-alpine
 POSTGRES_DEBIAN_TAG ?= 10.19
+SCHEMAHERO_TAG ?= 0.12.5
```

Comment:
update this to 0.12.7 if possible

Reply:
why not 0.12.7?

Reply:
version was updated in [separate pr](https://github.com/replicatedhq/kots/pull/2772/files#diff-bd6a4ce6cea38e97d64bca2cd1a6874c99a32186a0f6a7f307c51011b904b329).